app/ui-message/client/message.html

```diff
@@ -21,64 +21,65 @@
 			{{/if}}
 		{{/if}}
-		<button type="button" class="user user-card-message color-primary-font-color" data-username="{{msg.u.username}}" tabindex="1">{{getName}}{{#if showUsername}} <span class="message-alias border-component-color color-info-font-color">@{{msg.u.username}}</span>{{/if}}</button>
-		<span class="info border-component-color color-info-font-color">
-			{{#each role in roleTags}}
-				<span class="role-tag color-secondary-color border-component-color" data-role="{{role.description}}">{{role.description}}</span>
-			{{/each}}
-			{{#if isBot}}
-				<span class="is-bot color-secondary-color border-component-color">{{_ "Bot"}}</span>
-			{{/if}}
-			<time class="time" title='{{date}} {{time}}' datetime='{{date}} {{time}}'>{{time}}</time>
-			{{#if showTranslated}}
-				<span class="translated">
-					<i class="icon-language {{#if autoTranslateFetching}}loading{{/if}}" aria-label="{{_ "Translated"}}"></i>
-				</span>
-			{{/if}}
-			{{#if sentByEmail}}
-				<span class="sent-by-email">
-					<i class="icon-mail" title='{{_ "Message_sent_by_email"}}' aria-label='{{_ "Message_sent_by_email"}}'></i>
-				</span>
-			{{/if}}
-			{{#if edited}}
-				<span class="edited" title='{{_ "edited"}} {{_ "at"}} {{editTime}} {{_ "by"}} {{editedBy}}'>
-					<i class="icon-edit" aria-label="{{_ "Edited"}}"></i>
-					<button class="thumb thumb-small user-card-message" data-username="{{editedBy}}" tabindex="1">{{> avatar username=editedBy}}</button>
+		<div class="title">
+			<button type="button" class="user user-card-message color-primary-font-color" data-username="{{msg.u.username}}" tabindex="1">{{getName}}{{#if showUsername}} <span class="message-alias border-component-color color-info-font-color">@{{msg.u.username}}</span>{{/if}}</button>
+			<span class="info border-component-color color-info-font-color">
+				{{#each role in roleTags}}
+					<span class="role-tag color-secondary-color border-component-color" data-role="{{role.description}}">{{role.description}}</span>
+				{{/each}}
+				{{#if isBot}}
+					<span class="is-bot color-secondary-color border-component-color">{{_ "Bot"}}</span>
+				{{/if}}
+				<time class="time" title='{{date}} {{time}}' datetime='{{date}} {{time}}'>{{time}}</time>
+				{{#if showTranslated}}
+					<span class="translated">
+						<i class="icon-language {{#if autoTranslateFetching}}loading{{/if}}" aria-label="{{_ "Translated"}}"></i>
+					</span>
+				{{/if}}
+				{{#if sentByEmail}}
+					<span class="sent-by-email">
+						<i class="icon-mail" title='{{_ "Message_sent_by_email"}}' aria-label='{{_ "Message_sent_by_email"}}'></i>
+					</span>
+				{{/if}}
+				{{#if edited}}
+					<span class="edited" title='{{_ "edited"}} {{_ "at"}} {{editTime}} {{_ "by"}} {{editedBy}}'>
+						<i class="icon-pencil{{#if $neq editedBy msg.u.username}} error-color{{/if}}" aria-label="{{_ "Edited"}}"></i>
```

Comment:
Maybe we can use the SVG icon here.

Reply:
We will change all icons to fonts, so leave it.